_posts/2017-06-06-Spring-IO-2017-The-Spring-ecosystem.md

```diff
@@ -0,0 +1,318 @@
+---
+layout: post
+authors: [matthias_caryn]
+title: 'Spring IO 2017: The Spring Ecosystem'
+image: /img/spring-ecosystem/springEcosystemHeaderImageJPG.jpg
+tags: [Spring, Ecosystem, Cloud, Conference]
+category: Spring
+comments: true
+---
+
+
+> When I was at Spring IO back in May, I was intrigued by a presentation given by Jeroen Sterkens. 
+There he talked about the Spring Ecosystem in 50 minutes. 
+As only having 50 minutes, he could not focus on all the projects and I wanted to get a feel what Pivotal Software has to offer in all its glory by getting to know all the main projects.
+
+<span class="image left small"><img class="p-image" alt="Jeroen Sterken" src="/img/spring-ecosystem/jeroen-sterken.jpg"></span>
+
+Jeroen Sterken ([@jeroensterken](https://twitter.com/jeroensterken)) is a Java and Spring consultant from Belgium. He's a certified Spring instructor. He currently works at Faros Belgium. His slides of his talk _The Spring Ecosystem in 50 minutes_ can be found [here](https://www.slideshare.net/JeroenSterken1/the-spring-ecosystem-in-50-min).
```

Comment:
Make it one sentence, He's a certified Spring instructor and currently employed at Faros